Tasks/AzureRmWebAppDeployment/azurermwebappdeployment.ts

```diff
@@ -33,6 +33,9 @@ async function run() {
         var JSONFiles = tl.getDelimitedInput('JSONFiles', '\n', false);
         var xmlVariableSubstitution: boolean = tl.getBoolInput('XmlVariableSubstitution', false);
         var endPointAuthCreds = tl.getEndpointAuthorization(connectedServiceName, true);
+        var addWebConfig = tl.getBoolInput('AddWebConfig', false);
+        var appType = tl.getInput('AppType', false);
```

Comment:
work is in progress for adding app type input in editor extension. this is for testing purpose.